crates/symbolicator/src/services/download/sentry.rs

```diff
@@ -224,6 +224,7 @@ impl SentryDownloader {
                     FileType::WasmDebug | FileType::WasmCode => "wasm",
                     FileType::Breakpad => "breakpad",
                     FileType::SourceBundle => "sourcebundle",
+                    FileType::Usym => "usym",
```

Comment:
does sentry ignore unknown ones?  because it won't know about this yet.
it would be the logical thing to expect, but maybe best to double check

Reply:
I addressed this by skipping the usym filetype in this function for now.